Demo.xcodeproj/project.pbxproj

```diff
@@ -293,6 +293,7 @@
 			isa = XCBuildConfiguration;
 			buildSettings = {
 				ALWAYS_SEARCH_USER_PATHS = NO;
+				CLANG_ANALYZER_LOCALIZABILITY_NONLOCALIZED = YES;
```

Comment:
did this trigger some warning ?

Reply:
Yes, this is a new-in-xcode-10.2 thing, IIRC